common/lib/dependabot/shared_helpers.rb

```diff
@@ -151,7 +150,6 @@ def self.excon_defaults(options = nil)
       options ||= {}
       headers = options.delete(:headers)
       {
-        instrumentor: ActiveSupport::Notifications,
```

Comment:
I'm not sure what this was gaining us since I don't see us subscribing to any other notifications?

Reply:
Ah I see in dry-run we do have a subscribe.

Reply:
I'll see if there's another way to achieve what we had without ActiveSupport.